src/main/java/kitchenpos/tablegroup/validator/TableGroupValidator.java

```diff
@@ -0,0 +1,65 @@
+package kitchenpos.tablegroup.validator;
+
+import kitchenpos.common.constants.ErrorCodeType;
+import kitchenpos.order.domain.Order;
+import kitchenpos.order.domain.OrderTable;
+import kitchenpos.order.domain.OrderTables;
+import kitchenpos.order.port.OrderPort;
+import kitchenpos.order.port.OrderTablePort;
+import kitchenpos.tablegroup.domain.TableGroup;
+import kitchenpos.tablegroup.dto.TableGroupRequest;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+public class TableGroupValidator {
+
+    private static final int TABLE_GROUP_MIN_SIZE = 2;
+
+    private final OrderPort orderPort;
+
+    private final OrderTablePort orderTablePort;
+
+    public TableGroupValidator(OrderPort orderPort, OrderTablePort orderTablePort) {
+        this.orderPort = orderPort;
+        this.orderTablePort = orderTablePort;
+    }
+
+    public void makeTableGroup(TableGroupRequest request) {
```

Comment:
```suggestion
    public void makeTableGroup(TableGroup tableGroup) {
```
validator에서 dto보다 도메인 객체에 의존하는게 더 낫지않을까요? 🤔